lib/osf-components/addon/components/validated-input/base-component.ts

```diff
@@ -27,6 +27,7 @@ export default abstract class BaseValidatedInput extends Component {
     placeholder?: string;
     disabled: boolean = defaultTo(this.disabled, false);
     messagesShown: boolean = defaultTo(this.messagesShown, true);
+    ref!: BaseValidatedInput;
```

Comment:
Is this actually used? It's set below but I don't see it read.

Reply:
Nope. I *was* using that approach, but switched to binding just the reset action with `bindReset` and forgot to remove this. Thanks for the :eyes:.